docs/platform/enterprise-flex/data-plane.md

```diff
@@ -0,0 +1,452 @@
+When creating a new workspace:
+
+<details>
+  <summary>Request</summary>
+
+Send a POST request to /v1/workspaces/
+
+```bash
+curl -X POST "api.airbyte.com/v1/workspaces" \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/json" \
+  -d '{
+    "name": "My New Workspace",
+    "dataResidency": "auto"
+  }'
+```
+
+Include the following parameters in your request.
+
+| Body parameter  | Description                                               |
+| --------------- | --------------------------------------------------------- |
+| `name`          | The name of your workspace in Airbyte.                    |
+| `dataResidency` | A string with a region identifier you received in step 1. |
+
+For additional request examples, see [the API reference](https://reference.airbyte.com/reference/workspaces#/).
+
+</details>
+
+<details>
+  <summary>Response</summary>
+
+```json
+{
+  "workspaceId": "uuid-string",
+  "name": "workspace-name",
+  "dataResidency": "auto",
+  "notifications": {
+    "failure": {},
+    "success": {}
+  }
+}
+```
+
+</details>
+
+When updating a workspace:
+
+<details>
+  <summary>Request</summary>
+
+Send a PATCH request to /v1/workspaces/`{workspaceId}`.
+
+```bash
+curl -X PATCH "https://api.airbyte.com/v1/workspaces/{workspaceId}" \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/json" \
+  -d '{
+    "name": "Updated Workspace Name",
+    "dataResidency": "us-west"
+  }'
+```
+
+Include the following parameters in your request.
+
+| Body parameter  | Description                                               |
+| --------------- | --------------------------------------------------------- |
+| `name`          | The name of your workspace in Airbyte.                    |
+| `dataResidency` | A string with a region identifier you received in step 1. |
+
+For additional request examples, see [the API reference](https://reference.airbyte.com/reference/workspaces#/).
+
+</details>
+
+<details>
+  <summary>Response</summary>
+
+```json
+{
+  "workspaceId": "uuid-string",
+  "name": "updated-workspace-name",
+  "dataResidency": "region-identifier",
+  "notifications": {
+    "failure": {},
+    "success": {}
+  }
+}
+```
+
+</details>
+  </TabItem>
+</Tabs>
+
+## 4. Configure Kubernetes Secrets {#step-4}
+
+Your data plane relies on Kubernetes secrets to identify itself with the control plane.
+
+In step 5, you create a values.yaml file that references this Kubernetes secret store and these secret keys. Configure all required secrets before deploying your data plane.
+
+You may apply your Kubernetes secrets by applying the example manifests below to your cluster, or using kubectl directly. If your Kubernetes cluster already has permissions to make requests to an external entity via an instance profile, credentials aren't required. For example, if your Amazon EKS cluster has a sufficient AWS IAM role to make requests to AWS S3, you don't need to specify access keys.
```

Comment:
Can we include a breakdown of when we need a secrets manager, and when we don't? And if we support instance profiles / assuming a role? Or if we only support keys? 

I think this could land better if it's explained as per in the implementation guide (or in a similar format): https://airbyte-docs-rcqb653vw-airbyte-growth.vercel.app/platform/next/enterprise-setup/implementation-guide#configuring-external-connector-secret-management

The multi-region doc, where this is pulled from, assumed prior knowledge from reading the implementation guide for SME